.github/workflows/ci_linux_x64_clang_debug.yml

```diff
@@ -51,14 +51,15 @@ jobs:
         with:
           submodules: true
       - name: Install Python requirements
-        run: pip install -r ./runtime/bindings/python/iree/runtime/build_requirements.txt
+        run: python3 -m pip install -r ./runtime/bindings/python/iree/runtime/build_requirements.txt
       # Note: Not using ccache here. Debug builds need a lot of cache space
       # and caching only provides marginal build time improvements.
       - name: CMake - configure
         run: |
           cmake \
             -G Ninja \
             -B ${BUILD_DIR} \
+            -DPython3_EXECUTABLE="$(which python)" \
```

Comment:
We may want to switch this to
```suggestion
            -DPython3_EXECUTABLE="$(which python3)" \
```
if the [python-is-python3](https://packages.ubuntu.com/jammy/python-is-python3) package isn't installed and only the system provided version is available.

Reply:
Either could work. Leaving that for future cleanup.
![image](https://github.com/user-attachments/assets/dccfd3cb-1491-4981-ad81-fdff39c5f5d5)

Reply:
Well, `which python` is only in the markdown, but `which python3` seems to be what is actually used in the scripts. Anyway, fine for me if it works. (My Gentoo knows both, Ubuntu often does not..)